crawl-ref/source/beam.cc

```diff
@@ -4604,11 +4604,15 @@ void bolt::knockback_actor(actor *act, int dam)
     const int distance =
         (origin_spell == SPELL_FORCE_LANCE)
             ? 1 + div_rand_round(ench_power, 40) :
+        (origin_spell == SPELL_DIFFERENTIAL_EXPANSION)
+            ? 2 + div_rand_round(ench_power, 40) :
         (origin_spell == SPELL_CHILLING_BREATH) ? 2 : 1;
 
     const int roll = origin_spell == SPELL_FORCE_LANCE
-                     ? 7 + 0.27 * ench_power
-                     : 17;
+                     ? 7 + 0.27 * ench_power  :
+                     origin_spell == SPELL_DIFFERENTIAL_EXPANSION
+                     ? 10 + 0.24 * ench_power                     :
+                     17;
```

Comment:
Missing colon for the last in-line if statement (before the 17, see removed line).